src/Main/DetectiveGameMain.java

```diff
@@ -0,0 +1,132 @@
+
+    private static void displayCaseMenu(List<CaseFile> cases) {
+        System.out.println("╔══════════════════════════════════════════════════════════════════════════════════════════════╗");
+        System.out.println("║                                     SELECT A CASE TO INVESTIGATE                             ║");
+        System.out.println("╠══════════════════════════════════════════════════════════════════════════════════════════════╣");
+
+        if (cases.isEmpty()) {
+            System.out.println("║ No cases available. Please add cases to the 'cases' folder.                                    ║");
+        } else {
+            for (int i = 0; i < cases.size(); i++) {
+                System.out.printf("║ %d. %-85s ║%n", i + 1, cases.get(i).getTitle());
+            }
+        }
+
+        System.out.println("╚══════════════════════════════════════════════════════════════════════════════════════════════╝");
+    }
+
+    private static CaseFile selectCase(List<CaseFile> cases) {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.print("Enter the number of the case you wish to investigate: ");
+            try {
+                int choice = Integer.parseInt(scanner.nextLine());
+                if (choice > 0 && choice <= cases.size()) {
+                    return cases.get(choice - 1);
+                }
+            } catch (NumberFormatException ignored) {}
+            System.out.println("Invalid choice. Please select a valid case number.");
+        }
+    }
+
+    private static void startGame(CaseFile caseFile) {
+        // Initialize game components
+        Building building = BuildingExtractor.loadBuilding(caseFile);
+        SuspectExtractor.loadSuspects(caseFile, building);
+        GameObjectExtractor.loadObjects(caseFile, building);
+
+        // Initialize task list
+        TaskList taskList = new TaskList(caseFile.getTasks());
+
+        // Initialize characters
+        Detective detective = new Detective("Sherlock Holmes");
+        List<String> watsonHints = caseFile.getWatsonHints(); // Ensure CaseFile has this getter
+        DoctorWatson watson = new DoctorWatson(watsonHints);
+
+        // Initialize game context
+        GameContext context = new GameContext(building, detective, watson, new Journal(), taskList, caseFile); // Pass the selected case
+
+        // Ensure Watson's starting room matches the player's starting room
+        watson.setCurrentRoom(building.getCurrentRoom());
+        building.setWatson(watson);
+
+        // Display case invitation immediately after selecting a case
+        Letter letter = new Letter();
+        LetterExtractor.loadLetter(caseFile, letter);
+        letter.displayInvitation();
+        System.out.println("\nNow type 'start case' to begin the investigation.");
+
+        // Command loop
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.print("C:\\DetectiveGame> ");
```

Comment:
What does this mean?
What if i'm playing the game on macos or linux? there are no C drives on those operating systems